compute/cloud-client/src/test/java/compute/disks/DisksIT.java

```diff
@@ -317,6 +317,7 @@ public void testCreateReplicatedDisk()
             replicaZones, REGIONAL_REPLICATED_DISK, 100, DISK_TYPE);
 
     assertThat(status).isEqualTo(Status.DONE);
+    assertThat(status).isEqualTo(Status.DONE);
```

Comment:
Why is this assert statement duplicated here?

Reply:
It's a mistake. Thank you! Deleted